interfaces/dotnet/examples/SoundSpeed/Program.cs

```diff
@@ -0,0 +1,72 @@
+/*
+Cantera C# Examples
+Application example
+
+Computes the “equilibrium” and “frozen” sound speeds for a gas.
+*/
+
+using Cantera;
+
+Application.DataDirectories.AddAssemblyDirectory();
+
+Application.AddConsoleLogging();
+
+var phase = Application.CreateThermoPhase("gri30.yaml");
+var species = phase.Species;
+
+species.SetMoleFractions(("CH4", 1.00), ("O2", 2.20), ("N2", 7.52));
```

Comment:
For parity with the Python example, and to get the composition of air right, this should be:
```suggestion
species.SetMoleFractions(("CH4", 1.00), ("O2", 2.00), ("N2", 7.52));
```

Reply:
corrected